web-ui/src/main/java/com/tutrit/webclient/WebClientApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
```

Comment:
changes to `web-ui` module are out of the scope of this issue

Reply:
OK, I put it back